### DISCUSSION TITLE
academy/web-scraping-for-beginners/data-extraction/using-devtools

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1055] [Comments]
# academy/web-scraping-for-beginners/data-extraction/using-devtools

Learn how to use browser DevTools, CSS selectors, and JavaScript via the DevTools console to extract data from a website.

https://docs.apify.com/academy/web-scraping-for-beginners/data-extraction/using-devtools

**Comment by FlowExtractAPI** [comment]
great content
